test/commands/trigger_test.exs

```diff
@@ -1,13 +1,15 @@
 defmodule Cog.Test.Commands.TriggerTest do
-  use Cog.CommandCase, command_module: Cog.Commands.Trigger
+  use Cog.CommandCase, command_tag: :trigger
```

Comment:
Is there any benefit to splitting out tests into separate files, one per subcommand? It feels like there's a bit more magic in the setup than we might need.